printer/util.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package printer
+
+import (
+	"bytes"
+	"errors"
+	"os"
+
+	"golang.org/x/term"
+)
+
+func checkInteractiveTerminal() error {
+	fileInfo, err := os.Stdout.Stat()
+	if err != nil {
+		return err
+	}
+
+	if (fileInfo.Mode() & os.ModeCharDevice) == 0 {
+		return errors.New("this is not an interactive shell")
+	}
+
+	return nil
+}
+
+func termHeight(file *os.File) int {
+	_, h, err := term.GetSize(int(file.Fd()))
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Not sure how likely this is but seems a bit harsh to panic due to an error here. I'd expect the feature to be disabled.

Reply:
Fair enough 👍